src/sentry/reprocessing2.py

```diff
@@ -473,4 +542,10 @@ def get_progress(group_id):
     if info is None:
         logger.error("reprocessing2.missing_info")
         return 0, None
-    return int(pending), json.loads(info)
+
+    info = json.loads(info)
+    # Our internal sync counters are counting over *all* events, but the
+    # progressbar in the frontend goes until max_events. Advance progressbar
+    # proportionally.
+    pending = int(int(pending) * info["totalEvents"] / float(info.get("syncCount", 1)))
```

Comment:
This might be out-of-scope for this PR, but should we maybe just update the UI to reflect the number of events that are actually being touched?

Is there any other API client that relies on this information being a real event count, instead of a scaled down version?

Reply:
I think there's too many places that we'd have to redesign wording of. Besides the progressbar the same counter is used in the activity feed of an issue.